Enterprise-CMCS/managed-care-review
# [MCR-2539] CMS notified when a state is sent questions

## Summary

This PR creates a way for CMS users to be notified via email when someone from the same CMS division sends a question to a state. The state analysts for a state will be notified regardless of which CMS division the question came from.

#### Related issues

https://qmacbis.atlassian.net/browse/MCR-2539

#### Screenshots

<img width="1155" alt="Screenshot 2023-12-05 at 4 31 44 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/8137e725-5ae8-48ed-8756-0c46a4f8d583">


#### Test cases covered

-  the to address list includes state analysts regardless of which CMS division submitted the question
-  the to address list includes OACT emails when someone from OACT submitted the question
-  the to address list includes DMCP emails when someone from DMCP submitted the question
-  subject line is correct
-  the email itself includes relevant info (like link to a QA, name, email and division of the CMS user who submitted the question)
-  renders overall email

## QA guidance

- Login to the review app as a CMS user
- Find a submitted submission and click on the QA tab, upload a question
- Check the email, state analysts should be notified and the group DMCP is a DMCP user submitted the question, same for OACT
